src/FunctionalLimitedCollection.js

```diff
@@ -0,0 +1,128 @@
+import {FunctionalArrayCollection} from './FunctionalArrayCollection'
+
+/**
+ * Throw Error if add new item to collection and collection
+ * hold too many items.
+ *
+ * @since [*next-version*]
+ *
+ * @type {string}
+ */
+export const FLC_OVERFLOW_THROW = 'throw';
+
+/**
+ * Start adding item to the collection from start if
+ * there are to many items in the collection.
+ *
+ * @since [*next-version*]
+ *
+ * @type {string}
+ */
+export const FLC_OVERFLOW_START = 'start';
```

Comment:
It appears that currently, it works as treating all behaviour that is not "start" or "throw" as "append". The "append" behaviour needs to have a constant of its own. If you do that, it also makes sense to rename this one to `FLC_OVERFLOW_PREPEND`.

Reply:
I think `throw` should be default mode, not `append`. My fault.